File: application/models/Matiere.php
<?php
class Application_Model_Matiere extends Application_Model_Element_Abstract {
    protected $_idField = 'matid';
    protected  $_uniteid;
    protected $_matlib;
    protected $_matsuppr;
    protected $_matprix;
   
    public function getMatprix() {
        return $this->_matprix;
    }

    public function setMatprix($_matprix) {
        $this->_matprix = $_matprix;
    }

        public function getIdField() {
        return $this->_idField;
    }

    public function setIdField($_idField) {
        $this->_idField = $_idField;
    }

    public function getMatlib() {
        return $this->_matlib;
    }

    public function setMatlib($_matlib) {
        $this->_matlib = $_matlib;
    }

    public function getMatsuppr() {
        return $this->_matsuppr;
    }

    public function setMatsuppr($_matsuppr) {
        $this->_matsuppr = $_matsuppr;
    }
    public function getUniteid() {
    	return $this->_uniteid;
    }
    
    public function setUniteid($_uniteid) {
    	$this->_uniteid = $_uniteid;
    }    
}

?>

File: src/models/Matiere.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package models;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

/**
 *
 * @author overadmin
 */
@Entity
@Table(name="matiere")
public class Matiere {
    
    @Id
    @GeneratedValue(strategy= GenerationType.IDENTITY)
    private int matid;
    
    @Column(name="uniteid", columnDefinition="INTEGER", nullable=true)
    private int uniteid;
    
    @Column(name="matlib", columnDefinition="VARCHAR(100)", nullable=true)
    private String matlib;
    
    @Column(name="matsuppr", columnDefinition="BOOLEAN", nullable=true)
    private Boolean matsuppr;  

    public int getMatid() {
        return matid;
    }

    public void setMatid(int matid) {
        this.matid = matid;
    }

    public int getUniteid() {
        return uniteid;
    }

    public void setUniteid(int uniteid) {
        this.uniteid = uniteid;
    }

    public String getMatlib() {
        return matlib;
    }

    public void setMatlib(String matlib) {
        this.matlib = matlib;
    }

    public Boolean getMatsuppr() {
        return matsuppr;
    }

    public void setMatsuppr(Boolean mattsuppr) {
        this.matsuppr = mattsuppr;
    }
}
